activemodel/CHANGELOG.md

```diff
@@ -19,4 +19,9 @@
 
     *Jonathan Hefner*
 
+*   Add `touch` option to `#update_columns` and `#update_column` methods. When
```

Comment:
This should be in `activerecord/CHANGELOG.md`.

I would change it to something like
```
*  Add `touch` option to `#update_columns` and `#update_column` methods.

   Example:
   ```ruby
   # Will update :updated_at/:updated_on alongside :nice column.
   user.update_column(:nice, true, touch: true)
   ```
```